WordPress/gutenberg
# [RNMobile] Fix crash bringing gutenberg master to mobile

## Description


This patch makes the minimal adjustments to make the native version of gutenberg work with the latest change to the core data entities. Since we don't yet have networking support on mobile, the approach explored here is to preload the store with the required core data entities so it does not fail on load.

### Testing Instruction

- Checkout Gutenberg Mobile PR https://github.com/wordpress-mobile/gutenberg-mobile/pull/1366 and update submodule
- Start demo app `yarn start:reset` and `yarn android`
- Check that the app boots and that nothing is broken in the UI
- Check that the tests pass
